src/Components/DrillPage.js

```diff
@@ -59,9 +59,11 @@ export const DrillPage = props => {
     );
   };
 
-  navigation.setOptions({
-    headerRight: () => displayFavoriteButton(),
-  });
+  useLayoutEffect(() =>
```

Comment:
Why not using useEffect directly?

Reply:
stupid copy paste from https://reactnavigation.org/docs/navigation-prop/#setoptions

Reply:
Should we swith to useEffect then or keep it this way?

Reply:
From what I understand, useEffect is run asynchronously, potentially a few nanoseconds after the render, so it could cause a flash of non-modified header between the first render and the application of the effect on very slow device. whereas useLayoutEffect is run synchronously, ie not leaving time for a bad render to reach the screen.

So let's keep it this way.